Instagram/View Controllers/HomeViewController.m

```diff
@@ -0,0 +1,49 @@
+//
+//  HomeViewController.m
+//  Instagram
+//
+//  Created by Sarah Wang on 7/6/21.
+//
+
+#import "HomeViewController.h"
+#import "Parse/Parse.h"
+#import "LoginViewController.h"
+#import "AppDelegate.h"
+#import "SceneDelegate.h"
+
+@interface HomeViewController ()
+
+@end
+
+@implementation HomeViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+- (IBAction)didTapLogout:(id)sender {
+    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
+        // PFUser.current() will now be nil
+    }];
+    
+    SceneDelegate *myDelegate = (SceneDelegate *)self.view.window.windowScene.delegate;
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
+    LoginViewController *loginViewController = [storyboard instantiateViewControllerWithIdentifier:@"LoginViewController"];
```

Comment:
in cases where there is one of something I tend to name it without any delineators. ex: `controller` if there's a second controller added then you can rename this to `loginController` and have the other named similarly.